README.rst

```diff
@@ -41,6 +43,8 @@ Viewer0D
 
 * **Keithley_Pico**: Pico-Amperemeter Keithley 648X Series, 6430 and 6514
 * **Keithley2110**: Multimeter Keithley  2110
-
-
-
+* **Keithley27XX**: Keithley 27XX Multimeter/Switch System using SCPI commands (tested with models 2700 & 2701)
+This viewer aims to use different switching modules (=acquisition cards) such as module 7706 or 7708.
```

Comment:
adds these line in the last paragraph otherwise it will badly render once on the plugin_manager

Reply:
I separated the description and the detector name, is the new paragraph ok or do I need to create a complete new section to write this description ?